operator/webhook/introute/sync.py

```diff
@@ -262,6 +267,11 @@ def _generate_container_env_vars(parent) -> List[Mapping[str, str]]:
         if keystore_password_env := _get_keystore_password_env(tls):
             env_vars.append(keystore_password_env)
 
+    env_vars.append(_service_name_env_var(parent))
+
+    if additional_env_vars := parent["spec"].get("env"):
+        for additional_env_var in additional_env_vars:
+            env_vars.append( { "name": additional_env_var["name"], "value": additional_env_var["value"] } )
```

Comment:
:memo: nitpick:  you could shorten this block to
```suggestion
env_vars.extend(parent["spec"].get("env", []))
```